antifmt.sh

```diff
@@ -16,6 +16,8 @@ if [ -z "$*" ]; then
 	exit 1
 fi
 
+. ./common.sh
+validate_externals unzip unrar 7zr bunzip2 gunzip unxz tar pdftotext
```

Comment:
Warning: personal opinion coming up: :stuck_out_tongue: 

I dislike `.` for sourcing in scripts, as it's so easily glossed over and missed. `source` is much clearer imo... 

(Similarly, in scripts I'll generally be using `--long` instead of `-l` for clarity... but that's more off topic).

Reply:
Yes and no.
`source` is better but not `sh` unfortunately.
Some scripts are still `sh` and therefore this is more portable.

Reply:
Ah, I wasn't aware of that. Silly `sh`.... :disappointed:

Reply:
I'm fine with doing everything in bash but that is a separate issue

Reply:
Agreed.